### ISSUE TITLE
Hotspot Onboarding Free Location Assert

### ISSUE THREAD

**Issue by matthewcarlreetz** [helium/hotspot-app#129]
When adding a hotspot that has a prepaid location assertion, the UI states that I have insufficient balance.

**Comment by matthewcarlreetz** [comment]
![Screenshot_20210201-114419](https://user-images.githubusercontent.com/1760821/106498326-d364ee80-6484-11eb-8433-a66626fc51aa.png)

**Comment by matthewcarlreetz** [comment]
Related feedback...

![@JackeyTY - Discord 2021-02-17 at 12 11 08 PM](https://user-images.githubusercontent.com/1760821/108248513-9140db80-7119-11eb-92d2-583928061d1e.jpg)

**Comment by tyler-whitman** [comment]
I've tested on my account and the free assert flow works for me, I did hardcode it to true though as I actually don't have a free assert left on my hotspot.

So in terms of things failing. Maybe the call to load your balance failed and thats why it shows 0? 

Also the loadLocationFeeData failing is interesting. I am going to add more error checking and alerts so hopefully we can track this down because everything seems to work well for my account / hotspot.
